utils/CMakeLists.txt

```diff
@@ -25,6 +25,16 @@ target_link_libraries(kdbx-extract
                       ${GCRYPT_LIBRARIES}
                       ${ZLIB_LIBRARIES})
 
+add_executable(kdbx-merge kdbx-merge.cpp)
+target_link_libraries(kdbx-merge
+                      keepassx_core
+                      ${MHD_LIBRARIES}
```

Comment:
Here MHD lib are useless, also I would remove Qt5::Concurrent and Qt5::Widgets.
It works fine with just:
```
                      keepassx_core
                      Qt5::Core
                      ${GCRYPT_LIBRARIES}
                      ${ZLIB_LIBRARIES}
```